src/audio_transcription/transcription_nodes.py

```diff
@@ -464,16 +464,16 @@ def execute(self, audio, transcription_interval=8.0, accumulation_duration=8.0,
             Tuple containing transcribed text (empty string if not ready to output)
         """
         try:
-            # Parse audio input - expect (audio_data, sample_rate) from LoadAudioTensorStream
-            if isinstance(audio, tuple) and len(audio) == 2:
-                audio_data, sample_rate = audio
-            elif hasattr(audio, 'shape'):
-                # Fallback for direct numpy array (backward compatibility)
-                audio_data = audio
-                sample_rate = 16000  # Default Whisper-optimized rate
-                logger.debug("Using fallback audio format detection")
-            else:
-                logger.warning(f"Unexpected audio format: {type(audio)}, returning empty")
+            # Direct audio processing - sample_rate is now a separate parameter from ComfyUI
+            audio_data = audio
```

Comment:
Any reason to maintain backward compatibility here?

```suggestion
if isinstance(audio, tuple): logger.warning("Tuple audio format deprecated"); audio_data, sample_rate = audio else audio_data = audio
```

Reply:
I'll simplify so that sample rate always comes from the node input

Reply:
There may be some comfyui nodes that provide waveform data as a tuple, will check, but I don't think we necessarily need to support it. It's better to have a single source